### PR TITLE
Feature/blacklist 78

### PR DESCRIPTION
**Pull request by sebbalex**
Blacklists are needed to exclude individual repository that are not in line with our [guidelines](https://docs.italia.it/italia/developers-italia/policy-inserimento-catalogo-docs/it/stabile/approvazione-del-software-a-catalogo.html).

BL will be evaluated using commands `one` and `crawl`.
- `one`: whether `[repo url]` is present in one of indicated blacklist, crawler will exit immediately. Basically ignore all repository defined in list preventing the unauthorized loading in catalog. 
- `crawl`: crawler will try to match each repository URL in its list with the ones listed in blacklists and if it so it will print a warn log and skip all operation on it. Furthermore it will immediately remove the blacklisted repository from ES if it is present.

### CODE REVIEW

File: crawler/crawler/crawler.go [hunk 1]
```diff
@@ -144,7 +144,32 @@ func (c *Crawler) CrawlPublishers(publishers []PA) error {
...
+	// is listed.
+	// we should return the ones listed to crawl command
+	// and call deleteFromES if present
+	toBeRemoved := c.removeBlackListedFromRepositories(GetAllBlackListedRepos())
+
+	return toBeRemoved, c.crawl()
+}
+
+func (c *Crawler) removeBlackListedFromRepositories(listedRepos map[string]string) (toBeRemoved []string) {
+	temp := make(chan Repository, 1000)
```
Comment (libremente) [inline crawler/crawler/crawler.go:158]: I don't get this, could you please add a comment? Thank you

File: crawler/cmd/one.go [hunk 2]
```diff
@@ -50,15 +56,15 @@ func getPAfromWhiteList(repoURL string, args []string) (pa crawler.PA) {
...
+			log.Tracef("matching %s with %s", paWlRepo, repoURL)
 			if paWlRepo == repoURL {
 				log.Debugf("PA found in whitelist %+v", paWl)
 				return paWl
 			}
 		}
 		// looking into organizations
 		for _, paWlRepo := range paWl.Organizations {
-			log.Debugf("matching %s.* with %s", repoURL, paWlRepo)
+			log.Tracef("matching %s.* with %s", repoURL, paWlRepo)
```
[the remark this answers is not in the document]
Comment (sebbalex) [reply crawler/cmd/one.go:67]: good catch, probably it is a legacy for some different code.

File: crawler/crawler/crawler.go (hunk 1 above)
Comment (sebbalex) [reply crawler/crawler/crawler.go:158]: Basically this function is in charge to assign a new channel that discards repositories in blacklists.

File: crawler/cmd/one.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (sebbalex) [reply crawler/cmd/one.go]: ok, I figured it out: that was from whitelist implementation and it checks if provided repository URL for `one` command is in whitelist, in that specific line it looks in ORG if some of them match url provided to use its codiceIPA, such as:

`matching https://github.com/italia.* with https://github/italia/repo1`

**Pull request merged**